src/controllers/events.ts

```diff
@@ -0,0 +1,21 @@
+import Raven from 'raven'
+import { EventAttributes, Events, URLAttributes, UserAttributes } from '../db'
+
+export const createEvent = async (
+  url: URLAttributes,
+  req,
+  user: UserAttributes | undefined,
+) => {
+  try {
+    const event = Events.create({
+      code: url.code,
+      fromIP: req.headers['x-forwarded-for'] || req.connection.remoteAddress,
```

Comment:
Any better way of doing this? @championswimmer

Reply:
https://www.npmjs.com/package/request-ip this handles all cases